lib/identity/data/data_sources/remote_identity_data_source.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:typed_data';
 
 import 'package:http/http.dart';
 import 'package:polygonid_flutter_sdk/identity/data/dtos/rhs_node_dto.dart';
+import 'package:polygonid_flutter_sdk/identity/data/mappers/state_identifier_mapper.dart';
```

Comment:
No mapper in DS, they should be used by the repo and pass the result as param.

Reply:
Will leave the reorg of this for another task, because of the complexity of the impl